drv_helpers.c

```diff
@@ -522,11 +522,14 @@ void *drv_dumb_bo_map(struct bo *bo, struct vma *vma, uint32_t map_flags)
 
 int drv_bo_munmap(struct bo *bo, struct vma *vma)
 {
+	int ret = 0;
 	if (vma->cpu) {
 		free(vma->addr);
 		vma->addr = NULL;
 	}
-	return munmap(vma->addr, vma->length);
+	if (vma->addr != NULL)
+		ret = munmap(vma->addr, vma->length); 
```

Comment:
Check spaces/tab used, ideally it should be 4 spaces

Reply:
I have followed the indentation used before as reference. In this function, it was tab space that was used before, so i have followed the same.